packages/ember-cookies/src/clear-all-cookies.js

```diff
@@ -1,10 +1,11 @@
 import { assert } from '@ember/debug';
```

Comment:
moved the file to new location to keep existing import path `ember-cookies/clear-all-cookies` work